### DISCUSSION TITLE
CI/CD

### DISCUSSION THREAD

**Discussion by DaddaAdam** [DaddaAdam/Spring-Reddit#30] [Q&A] [answered]
Wouldn't Jenkins be better suited than GitHub actions? Considering the complexity of the build and deployment workflow and the fact that Jenkins has built-in support for tools such as Kubernetes

**Comment by xenedium** [comment, marked as answer]
Well both are great CI tools the main reason that made us choose Github actions is that it integrates seamlessly with the Github ecosystem (PRs, Issues, repos etc) and is also easy to setup.
It also provides built in for Kubernetes, including actions for deploying and managing Kubernetes clusters.

Also if you care about security or  the integrity of your workflow you can use self-hosted runners.
